packages/core/src/Stores/traders-hub-store.js

```diff
@@ -176,35 +183,28 @@ export default class TradersHubStore extends BaseStore {
         return this.is_demo && low_risk;
     }
 
-    get account_flag() {
-        const { is_logged_in, upgradeable_landing_companies } = this.root_store.client;
-
-        if (is_logged_in) {
-            if (this.is_demo && upgradeable_landing_companies?.includes('svg')) {
-                return 'cr_demo_content';
-            } else if (
-                upgradeable_landing_companies.includes('svg') &&
-                upgradeable_landing_companies.includes('maltainvest') &&
-                !this.is_demo
-            ) {
-                if (this.is_eu_user) return 'low_risk_cr_eu_content';
-                return 'low_risk_cr_non_eu_content';
-            } else if (
-                upgradeable_landing_companies.length === 1 &&
-                upgradeable_landing_companies.includes('maltainvest')
-            ) {
-                if (this.is_demo) return 'eu_demo_content';
-                return 'eu_real_content';
-            } else if (
-                upgradeable_landing_companies.length === 1 &&
-                upgradeable_landing_companies.includes('svg') &&
-                !this.is_eu_user
-            ) {
-                return 'high_risk_cr_content';
-            }
-            return null;
+    get content_flag() {
+        const { is_logged_in, landing_companies, residence } = this.root_store.client;
+        const { financial_company, gaming_company } = landing_companies;
+
+        if (!is_logged_in) return '';
+        if (!gaming_company?.shortcode && financial_company?.shortcode === 'maltainvest') {
+            if (this.is_demo) return ContentFlag.EU_DEMO;
+            return ContentFlag.EU_REAL;
+        } else if (financial_company?.shortcode === 'maltainvest' && gaming_company?.shortcode === 'svg') {
+            if (this.is_eu_user) return ContentFlag.LOW_RISK_CR_EU;
+            return ContentFlag.LOW_RISK_CR_NON_EU;
```

Comment:
@yashim should'nt we check if the account is demo before assigning the content_flag LOW_RISK_CR_EU and LOW_RISK_CR_NON_EU? 

if demo account we can assign CR_DEMO since contents are same for both low risk and high risk CR demo